### PR TITLE
Enable ephysics on windows

### PR DESCRIPTION
**Pull request by cauebs**
It appears that it just works!

### CODE REVIEW

**Comment by cauebs** [comment]
Oh, accidentally removed some trailing whitespace again. But it's for the better!

File: meson.build [hunk 1]
```diff
@@ -251,7 +251,7 @@ config_h.set_quoted('PACKAGE_NAME', meson.project_name())
 config_h.set_quoted('PACKAGE_BIN_DIR', dir_bin)
 config_h.set_quoted('PACKAGE_LIB_DIR', dir_lib)
 config_h.set_quoted('PACKAGE_SRC_DIR', meson.source_root() / '')
-config_h.set_quoted('PACKAGE_BUILD_DIR', meson.current_build_dir() / '') 
+config_h.set_quoted('PACKAGE_BUILD_DIR', meson.current_build_dir() / '')
```
Comment (JPTIZ) [inline meson.build:254]: I am 100% sure it works because of this change.

**Comment by Coquinho** [comment]
strangely native-windows and this are building the same amount of files is it ok?

**Comment by JPTIZ** [comment]
> strangely native-windows and this are building the same amount of files is it ok?

Nope:

image

**Comment by JPTIZ** [comment]
OH right, of course, see `configure.bat`:

```batch
    set MESONFLAGS=^
            -Dopenssl_dir="%OPENSSL_DIR:"=%"^
            -Dcrypto=openssl^
            -Dnls=false^
            -Dsystemd=false^
            -Dglib=false^
            -Dgstreamer=false^
            -Ddbus=false^
            -Daudio=false^
            -Davahi=false^
            -Dv4l2=false^
            -Delua=false^
            -Dx11=false^
            -Dphysics=false^         <<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<
            -Deeze=false^
            -Dpulseaudio=false^
            -Dharfbuzz=false^
            -Dfribidi=false^
            -Dfontconfig=false^
            -Dedje-sound-and-video=false^
            -Dlibmount=false^
            "-Devas-loaders-disabler=gst,pdf,ps,raw,svg,rsvg,xcf,bmp,dds,eet,generic,gif,ico,jp2k,pmaps,png,psd,tga,tgv,tiff,wbmp,webp,xpm,json"^
            -Dopengl=none^
            "-Demotion-loaders-disabler=gstreamer1,libvlc,xine"^
            -Dbuild-tests=true^
            -Dbuild-examples=false^
            -Dbindings=^
            --buildtype=debug^
            --native-file native-file-windows.txt
```

**Comment by cauebs** [comment]
Well, it seems we're missing the `bullet` external dependency. Any pointers?

**Comment by joaoantoniocardoso** [comment]
> bullet

https://github.com/bulletphysics/bullet3 it seems simple to get.

**Comment by cauebs** [comment]
ephysics seems to be optional and I'll reopen this if and when we actually need to address it.

**Pull request closed without merging**